content/docs/command-reference/types.md

```diff
@@ -10,15 +12,16 @@ usage: mlem types [-h]
                   [abc] [sub_type]
 
 arguments:
-  [abc]            Subtype to list implementations. List subtypes if not
-                   provided.
+  [abc]            Subtype to list implementations.
+                   List subtypes if not provided.
   [sub_type]       Type of `meta` subtype
 ```
 
 ## Description
 
-This command can be used to see all available MLEM classes, or to list the
-different implementations available for a specific `SUB_TYPE` (argument).
+This command can be used to see all available MLEM object types (classes), or to
```

Comment:
Still not sure about the use of term "classes" here TBH. WDYT @aguschin ? 🙂

Reply:
They're Python classes indeed, but that's implementation detail. Still, may be good to tell this so people will start understanding MLEM a bit deeper - this will help to understand right abstractions maybe? Not sure.

Reply:
I don't think that just throwing the word _class_ will improve understanding. Probably doesn't hurt either but IMO it's better to use consistent terminology (we say "types" everywhere else rn).